proxy/src/auth/credentials.rs

```diff
@@ -37,10 +41,10 @@ impl ClientCredentials {
 
 #[derive(Debug, Error)]
 pub enum ProjectNameError {
-    #[error("SNI is missing, please upgrade the postgres client library")]
+    #[error("SNI info is missing. EITHER please upgrade the postgres client library OR pass the project name as a parameter: '..&options=project:<project name>..'.")]
```

Comment:
I'm not sure we should capitalize those words, but we have neither a formal procedure for figuring out texts like this nor a dedicated tech writer anyway, so I'm not insisting.

Reply:
I capitalized them as 'command' words. I don't have a strong preference either.